### PR TITLE
Wait for mount to mount

### PR DESCRIPTION
**Pull request by sblack4**

### CODE REVIEW

**Review by jjduverge — APPROVED** [review]

File: defaults/main.yml [hunk 1]
```diff
@@ -62,6 +62,7 @@ iqserver_rut_logout_url:
 
 # Base directory that iqserver is installed in. The module will create
 # an untarred dir, a current symlink, and a sonatype-work dir.
+wait_for_sonatype-work_mount: true
```
Comment (sdickenson) [inline defaults/main.yml:65]:
```suggestion
wait_for_sonatypework_mount: true
```

File: templates/etc.systemd.system.iqserver.service.j2 [hunk 2]
```diff
@@ -1,7 +1,12 @@
 # {{ ansible_managed }}
 [Unit]
 Description=Nexus IQ Service
+{% if wait_for_sonatype-work_mount %}
```
Comment (sdickenson) [inline templates/etc.systemd.system.iqserver.service.j2:4]:
```suggestion
{% if wait_for_sonatypework_mount %}
```

**Review by sdickenson — APPROVED** [review]
Mostly pedantic, but I hate mixing underscores and dashes. Not going to hold up approval for it though.

**Comment by sdickenson** [comment]
❤️

**Pull request merged**